de-lib/src/main/java/org/iplantc/de/pipelines/client/presenter/PipelineViewPresenter.java

```diff
@@ -160,14 +154,14 @@ private void initAppsView() {
         appSelectView = new AppSelectionDialog();
         appSelectView.setPresenter(this);
 
-        initAppsDragHandlers(appsPresenter.getAppsDragSources());
-        initPipelineBuilderDropHandler(view.getBuilderDropContainer());
+       // initAppsDragHandlers(appsPresenter.getAppsDragSources());
+      //  initPipelineBuilderDropHandler(view.getBuilderDropContainer());
```

Comment:
I have commented out logic for drag-n-drop and left in place most of the code related to WYSIWYG pipeline editor. Just in case...